### PR TITLE
Complete migration out of SDK.

### PR DESCRIPTION
**Pull request by erikcorry**
Adds some tests that were still in the SDK.
Fixes some imports so we get the non-SDK versions.

### CODE REVIEW

**Comment by erikcorry** [comment]
For now the Histogram tests are disabled.  Histogram will get its own repo at some point, when they can be reenabled.

File: tests/pbm_test.toit [hunk 1]
```diff
@@ -0,0 +1,135 @@
+// Copyright (C) 2020 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+import expect show *
+import pixel_display.two_color show *
+
+toit_logo := ByteArray 329: [
```
Comment (floitsch) [inline tests/pbm_test.toit:8]: Maybe do a follow-up PR, that changes this to `#[`.

File: tests/texture_test_slow.toit [hunk 2]
```diff
@@ -129,6 +129,7 @@ barcode_factory version code x y orientation:
       return true_color.BarCodeEan13 code x y transform
   unreachable
 
+/*
```
Comment (floitsch) [inline tests/texture_test_slow.toit:132]: Add a TODO with a link to an issue?
(unless you plan to fix this within the next days).

**Review by floitsch — APPROVED** [review]
LGTM.

**Pull request merged**